ch2.txt

```diff
@@ -30,6 +23,11 @@ CHAPTER 2. WAGING WAR
 
 16. Now in order to kill the enemy, our men must be roused to anger; that there may be advantage from defeating the enemy, they must have their rewards. 
 
+asdf
+asdf
+asdf
+adsf
+
```

Comment:
We snipped the top and bottom of this document so we can focus on the review comment here.